site/content/en/docs/handbook/accessing.md

```diff
@@ -53,13 +53,10 @@ Services of type `NodePort` can be exposed via the `minikube service <service-na
 3. Check Node Port
 
     ```shell
-    kubectl get svc
-    ```
-    <pre>
-    $ kc get svc
-    AME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    $ kubectl get svc
+    NAME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
     hello-minikube1   NodePort    10.100.238.34   <none>        8080:31389/TCP   3s
-    </pre>
```

Comment:
Thanks for the fix, kc was actually my alias for kubectl lol

however why you are removing /pre?

Reply:
Hello! I'm just trying to align to e.g. this below:
```
    ```shell
    $ ps -ef | grep docker@127.0.0.1
    ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -N docker@127.0.0.1 -p 55972 -i /Users/FOO/.minikube/machines/minikube/id_rsa -L TUNNEL_PORT:CLUSTER_IP:TARGET_PORT
    ```
```
Also it looked like it is duplicating the command.

I did not gave it too much of recommended style read / thought :)

Reply:
It's already in a code block so I don't think `<pre>` is needed